src/include/txn.i

```diff
@@ -851,24 +851,20 @@ __wt_txn_read_upd_list(
         if (type == WT_UPDATE_RESERVE)
             continue;
 
+        if (type == WT_UPDATE_TOMBSTONE && F_ISSET(&cbt->iface, WT_CURSTD_IGNORE_TOMBSTONE) &&
```

Comment:
This deserves a big comment. The one you removed was very out of date, but this code is subtle. I think something like:

If configured to ignore tombstones, record the timestamps from the tombstone in the returned value so that callers will know that there was a tombstone and what the time point on it was.

I think the flag is poorly named - it's actually asking the check to squash the tombstone down onto the previous update that is being returned. I can't think of a better name for now, so I'm not going to suggest renaming `WT_CURSTD_IGNORE_TOMBSTONE`, but I think we should do that in the future.

Reply:
Fixed in 6f0ddd7.